### PR TITLE
[EPT-1715] add list layout in prep for modal

### PR DESCRIPTION
**Pull request by shanthru**
This PR:

Adds list Layout and removes List View Top Panel.
Provides better layout for incorporating the new page modal

See https://github.com/sailthru/php/pull/5229 for data

### CODE REVIEW

**Review by Andrew-Gaffney — APPROVED** [review]
Looks good to me. One question is the list meant to/does it support sorting in anyway? Or is that something handled by stui-lists?

**Comment by shanthru** [comment]
> Looks good to me. One question is the list meant to/does it support sorting in anyway? Or is that something handled by stui-lists?

There is sorting available from the STUI table. It is currently set on Page name, Category and Last updated. Although looking at it now, the sort on Page name looks off. I'll look into updating that.

There was some talk of sorting coming from the backend so I'm not sure if that will eventually be replaced.

File: src/ListView/ListView.jsx [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
-import * as pagesApi from "../core/api";
+import * as pagesApi from "../core/pagesApi";
 
 function ListView() {
   const [loading, setLoading] = useState(true); // load from the start to prevent blip
   const [pageMap, setPageMap] = useState({});
+  const pagesData = useMemo(
+    () => [...Object.values(pageMap)],
+    [Object.values(pageMap)]
+  );
```
Comment (shanthru) [inline src/ListView/ListView.jsx:13]: Updated per this [comment](https://github.com/sailthru/hosted-pages-frontend/pull/8#discussion_r1042332631) in a previous PR

**Comment by shanthru** [comment]
closed for https://github.com/sailthru/hosted-pages-frontend/pull/12

**Pull request closed without merging**